com.unity.render-pipelines.core/Editor/ProjectorEditor.cs

```diff
@@ -0,0 +1,23 @@
+using UnityEngine;
+using UnityEngine.Rendering;
+
+namespace UnityEditor.Rendering
+{
+    [CustomEditorForRenderPipeline(typeof(Projector), typeof(RenderPipelineAsset))]
+    [CanEditMultipleObjects]
+    class ProjectorEditor : Editor
+    {
+        const string k_Message = "The active render pipeline does not support the Projector component. If using HDRP, use the Decal Projector component instead.";
```

Comment:
should it be a GUI_Content to support localization? i am not sure how this works

Reply:
@alex-vazquez flagged this for me as well. I looked into it and initially thought localization is not supported by the API, but looking back into it I found some use cases from the main editor codebase that do exactly that. I'll fix this.